### ISSUE TITLE
1.5.0: One test fails

### ISSUE THREAD

**Issue by yurivict** [Huluti/Curtail#142]
```
===>  Testing for curtail-1.5.0
===>   curtail-1.5.0 depends on file: /usr/local/bin/python3.9 - found
[  0% 1/1] /usr/local/bin/meson test --no-rebuild --print-errorlogs
1/3 Validate desktop file          OK              0.09s
2/3 Validate schema file           OK              0.06s
3/3 Validate appstream file        FAIL            0.31s   exit status 1
>>> MALLOC_PERTURB_=115 /usr/local/bin/appstream-util validate data/com.github.huluti.Curtail.appdata.xml
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― ✀  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
stdout:
data/com.github.huluti.Curtail.appdata.xml: FAILED:
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen1.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen1.png]
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen2.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen2.png]
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen3.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen3.png]
stderr:
Validation of files failed
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――


Summary of Failures:

3/3 Validate appstream file FAIL            0.31s   exit status 1

Ok:                 2   
Expected Fail:      0   
Fail:               1   
Unexpected Pass:    0   
Skipped:            0   
Timeout:            0   

Full log written to /usr/ports/graphics/curtail/work/Curtail-1.5.0/_build/meson-logs/testlog.txt
FAILED: meson-internal__test 
/usr/local/bin/meson test --no-rebuild --print-errorlogs
ninja: build stopped: subcommand failed.
```

clang-14
FreeBSD 13.1

**Comment by Huluti** [comment]
@bertob should I remove padding from screenshots?

**Comment by bertob** [comment]
Not sure what you mean by padding in the context of screenshots

**Comment by Huluti** [comment]
Seems to be the errors reported here:

```
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen1.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen1.png]
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen2.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen2.png]
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen3.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/Huluti/Curtail/master/data/screenshots/screen3.png]
```

**Comment by bertob** [comment]
Huh, not sure what that means. Does it still happen if you just retake the screenshot?

**Comment by Huluti** [comment]
Seems to be related to this:
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/162
https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3143
https://gitlab.gnome.org/GNOME/gtk/-/issues/5093

**Comment by Huluti** [comment]
Should be fixed in ab0e268bbb4c30900daf30ff255db243baa82465.
